move/sources/hongbao.move

```diff
@@ -16,6 +16,7 @@ module addr::hongbao {
     use aptos_framework::timestamp;
 
     const YEAR_IN_SECONDS: u64 = 31536000;
+    const MAX_ENVELOPES: u64 = 8888;
```

Comment:
this shouldn't matter in practice, no?

Reply:
it does right? the larger the hongbao the larger the amount transactions that can conflict if tps is really high